.reuse/dep5

```diff
@@ -8,7 +8,7 @@ Files:
     .dependabot/config.yml
```

Comment:
This line's not required anymore. Otherwise LGTM

Reply:
Right! Done.